packages/builder/src/pages/builder/portal/settings/ai/index.svelte

```diff
@@ -0,0 +1,171 @@
+<script>
+  import { onMount } from "svelte"
+  import {
+    Button,
+    Layout,
+    Heading,
+    Body,
+    Helpers,
+    Divider,
+    notifications,
+    Modal,
+    Tags,
+    Tag,
+  } from "@budibase/bbui"
+  import { admin, licensing } from "stores/portal"
+  import { API } from "api"
+  import AIConfigModal from "./ConfigModal.svelte"
+  import AIConfigTile from "./AIConfigTile.svelte"
+
+  const ConfigTypes = {
+    AI: "ai",
```

Comment:
What other options do we anticipate in here?

Reply:
None, was just to avoid putting the string directly